custom_components/opnsense/update.py

```
@@ -263,71 +204,79 @@
+            )
+        self.async_write_ha_state()
 
-        except:
-            return None
-        return summary
+    async def async_release_notes(self) -> str | None:
+        """Return the release notes of the latest version."""
+        return self._release_notes
 
-    def install(self, version=None, backup=False) -> None:
+    async def async_install(self, version=None, backup=False) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)